tests/common/plugins/sanity_check/__init__.py

```diff
@@ -119,8 +130,51 @@ def do_checks(request, check_items, *args, **kwargs):
     return check_results
 
 
+@pytest.fixture(scope="module", autouse=True)
+def log_custom_msg(request):
```

Comment:
what if errors are raise during other fixtures?
Will the logic after yield always run?
If not, we need try/final clause

Reply:
Confirmed early return, and exception raised from sanity_check will not skip yield for log_custom_msg. Teardown always happened when I tested with early return and exception raised manually.

Reply:
![image](https://github.com/user-attachments/assets/8ab1af64-e601-444f-84c4-12038f387cef)

Reply:
thanks!